rmw_fastrtps_cpp/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ find_package(ament_cmake REQUIRED)
 find_package(fastrtps_cmake_module REQUIRED)
 find_package(fastcdr REQUIRED CONFIG)
 find_package(fastrtps REQUIRED CONFIG)
+find_package(FastRTPS REQUIRED MODULE)
```

Comment:
I just find it weird that a package trying to use FastRTPS now needs to find three different items:

* our custom package `fastrtps_cmake_module`
* the upstream packages `fastcdr` and `fastrtps`
* our custom module file from `FastRTPS`

One line  (or max two lines) should be enough - that was the idea of having `fastrtps_cmake_module` in the first place: making it easier to use the upstream packages.

Anyway I see why finding the module is necessary for the new variable `FastRTPS_LIBRARIES` and this can be merged (after CI). I just think this is developing in the wrong direction but I sadly don't have the time to work on fixing this atm.